storage/internal/apiv1/storage_client_example_test.go

```diff
@@ -0,0 +1,1046 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by protoc-gen-go_gapic. DO NOT EDIT.
+
+package storage_test
+
+import (
+	"context"
+
+	storage "cloud.google.com/go/storage/internal/apiv1"
+	storagepb "google.golang.org/genproto/googleapis/storage/v1"
+)
+
+func ExampleNewClient() {
+	ctx := context.Background()
+	c, err := storage.NewClient(ctx)
+	if err != nil {
+		// TODO: Handle error.
+	}
+	defer c.Close()
+
+	// TODO: Use client.
+	_ = c
+}
+
+func ExampleClient_DeleteBucketAccessControl() {
```

Comment:
Out of curiosity, have these examples actually proven useful and/or been published in docs for any product?

Reply:
They GoDoc examples so they are included in the package [examples](https://pkg.go.dev/cloud.google.com/go/language/apiv1#pkg-examples) and with each [method doc](https://pkg.go.dev/cloud.google.com/go/language/apiv1#example-Client.AnalyzeEntities).

The snippetgen project generates stuff for the cloud site and tbp@ would be your PoC there.